tests/jballerina-unit-test/src/test/resources/test-src/statements/matchstmt/structured_error_match_patterns.bal

```diff
@@ -329,16 +329,28 @@ function testErrorConstReasonMatchPattern() returns string {
 
 
 type ER error<string, ErrorData>;
-
 function testIndirectErrorMatchPattern() returns string {
     ER err1 = error("Error Code", message = "Msg");
     match err1 {
         ER ( message = m, ...var rest) => {
             return <string>m;
         }
-        error(reason, ...var rest) => {
+        error(var reason, ...var rest) => {
             return reason;
         }
     }
     return "Default";
 }
+
+public function testErrorMatchWihtoutReason() returns string {
+    string|error se = error("test reason", message="error detail message");
+
+    match se {
```

Comment:
Does the wild card match pattern work for the error reason? If so, shall we add a test case for that as well?

Reply:
I think it's already there https://github.com/ballerina-platform/ballerina-lang/blob/master/tests/jballerina-unit-test/src/test/resources/test-src/statements/matchstmt/structured_error_match_patterns.bal#L151